PMNS_LIV.h

```diff
@@ -21,52 +27,31 @@ namespace OscProb {
     PMNS_LIV();          ///< Constructor
     virtual ~PMNS_LIV(); ///< Destructor
 
-    /// Set any given LIV parameter
-    virtual void SetaT(int flvi, int flvj, double val, double phase);
-    virtual void SetcT(int flvi, int flvj, double val, double phase);
-
-    /// Set the LIV parameters all at once
-    void SetLIV(double aT_ee,     double aT_mumu,      double aT_tautau,
-            double aT_emu,     double aT_etau,      double aT_mutau,
-            double cT_ee,     double CT_mumu,      double CT_tautau,
-            double cT_emu,     double cT_etau,      double cT_mutau,
-            double delta_aT_emu=0, double delta_aT_etau=0, double delta_aT_mutau=0,
-	    double delta_cT_emu=0, double delta_cT_etau=0, double delta_cT_mutau=0);
-
-    //Set diagonal LIV pars
-    virtual void SetaT_ee    (double a); ///< Set eps_ee parameter
-    virtual void SetaT_mumu  (double a); ///< Set eps_mumu parameter
-    virtual void SetaT_tautau(double a); ///< Set eps_tautau parameter
```

Comment:
Removing these functions will break some code from people who may have been using LIV with only 4 dimensions. Let's make sure to provide these as alternative definitions when dim<=4.

I suggest you collect these at the end of the cxx implementation file and separated by some comments to make it clear these are there mostly for backwards compatibility.